features/ios/ios_sessions.feature

```diff
@@ -51,6 +45,6 @@ Feature: iOS smoke tests for sessions
         And the error payload field "events.0.user.id" equals the stored value "automated_user_id"
         And the error payload field "events.0.device.id" equals the stored value "automated_user_id"
         And the event "session.id" is not null
-        And the error payload field "events.0.session.id" equals the stored value "automated_session_id"
+        #And the error payload field "events.0.session.id" equals the stored value "automated_session_id"
```

Comment:
Are these not equal anymore?

Reply:
Fixed, i forgot to switch it to use the new ignore case comparison step 👍